lib/src/sdk.ts

```diff
@@ -236,12 +236,16 @@ export class ExchangeSDK {
         tokenCurrency: toNewTokenId,
       })
       .catch(async (error: Error) => {
+        let swapStep = "";
```

Comment:
functional style nitpicking : I would prefer a 

```
const swapStep = swapStep(error)
...
function swapStep(error) {
        if ((error as CompleteExchangeError).step) {
          return (error as CompleteExchangeError).step;
        } else if (error.name === "DisabledTransactionBroadcastError") {
          return "SIGN_COIN_TRANSACTION";
        } else {
          return "UNKNOWN_STEP";
        }
}
```

let's have smaller functions and "const" vs "let", even for simple stuff.